prac2-pages/chhotu/lib/main.dart

```diff
@@ -0,0 +1,135 @@
+import 'package:flutter/material.dart';
+
+void main() => runApp(const MaterialApp(
+      home: Project(),
+    ));
+
+class Project extends StatefulWidget {
+  const Project({super.key});
+
+  @override
+  State<Project> createState() => _ProjectState();
+}
+
+class _ProjectState extends State<Project> {
+  bool value = false;
```

Comment:
It is a good practice to make your instance variables private. :exclamation: 

Kindly do make the change in the next commits